src/Microsoft.DotNet.Cli.Utils/project.json

```diff
@@ -10,6 +10,10 @@
     },
     "frameworks": {
         "net451": {
+            "imports": ["net46"],
```

Comment:
I think that means exactly what it sounds like.  If you want to target .NETFramework v4.5.1, you can't use the System.AppContext package. I would think the options are:
- Don't use System.AppContext
- or Microsoft.DotNet.Cli.Utils (and anything that references it) need to be bumped up to target net46.